trunk/SUAVE/Components/Energy/Networks/Battery_Propeller.py

```diff
@@ -186,12 +186,11 @@ def evaluate_thrust(self,state):
         conditions.propulsion.propeller_tip_mach    = (R*rpm)/a
         
         # Create the outputs
-        F    = self.number_of_engines * F * [np.cos(self.thrust_angle),0,-np.sin(self.thrust_angle)]      
-        mdot = np.zeros_like(F)
-
-        F_mag = np.atleast_2d(np.linalg.norm(F, axis=1)/Units.lbs) # lb   
+        F                                           = num_engines* F * [np.cos(self.thrust_angle),0,-np.sin(self.thrust_angle)]      
+        mdot                                        = np.zeros_like(F) 
+        F_mag                                       = np.atleast_2d(np.linalg.norm(F, axis=1)/Units.lbs) # lb   
```

Comment:
I just noticed this, your disc loading is in lb/ft^2. This isn't SI. I suggest we change this.

@timdmacdo thoughts?

Reply:
I used imperial because most of the literature use it. If you think it would be confusing I can use SI

Reply:
I think you need to put this into SI. If you do that I'll approve it. You can change it to US in the plotting script if you'd like.

I don't like setting a precedent of keeping internal data structures in mixed units, it's too dangerous.

Reply:
Yes I agree let's use SI. It often seems weird for individual values (sfc is another one) but not having consistent units almost always causes problems later.